Gemfile

```diff
@@ -42,6 +42,7 @@ gem "govuk-components", "~> 5.11.1"
 gem "govuk_design_system_formbuilder", "~> 5.11.0"
 gem "high_voltage"
 gem "httparty"
+gem "image_processing"
```

Comment:
From the commit this was added on. This gem seems to add inline PDF support for the attachments?
I don't find any explicit usage/call. 

It may be worth a comment to avoid the risk of a future "what is this for?, nothing calls it, 🪓 it" and then a random not explicitly tested feature breaks silently.